apps/client/src/app/pages/portfolio/activities/import-activities-dialog/import-activities-dialog.component.ts

```diff
@@ -39,7 +43,39 @@ export class ImportActivitiesDialog implements OnDestroy {
     this.dialogRef.close();
   }
 
-  public onImport() {
+  public async onImportActivities() {
+    try {
+      await this.importActivitiesService.importSelectedActivities(
```

Comment:
Can you add the following snack bar before actually importing?

```
this.snackBar.open('⏳ ' + $localize`Importing data...`);
```

Reply:
Done